### DISCUSSION TITLE
Highlight specific lines of code in a code block

### DISCUSSION THREAD

**Discussion by acheddir** [cotes2020/jekyll-theme-chirpy#1772] [Q&A] [answered]
### Checklist

- [x] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Built from `jekyll-theme-chirpy`

### Description

Is it possible to highlight specific lines in a fenced code block, just like in the image below

![image](https://github.com/cotes2020/jekyll-theme-chirpy/assets/3649139/b3636f1a-2d3f-49c9-8d3f-5aff3066e861)

### Operations you have already tried

Since Jekyll is using rouge, I tried adding the following at the end of my code block, with no success :
```md
{: highlight_lines=[2, 4]}
```
Maybe I'm doing it wrong, much help is appreciated 🙏🏼

**Comment by NichtsHsu** [comment, marked as answer]
Jekyll currently only supports code blocks generated from `{% highlight %}` to highlight specific lines. However, this syntax is not compatible with the Chirpy theme.

That is to say, currently do not have a native method to achieve this requirement. You must handle it yourself - writing a jekyll plug-in, or writing a js script, or something else.
